### ISSUE TITLE
Missing path/executable for gfserver

### ISSUE THREAD

**Issue by ycl6** [a-bioinformatician/BreaKmer#27]
Hi,

I placed blat, gfclient and gfserver in the same location after I downloaded the executable online, however, breakmer can only detect blat but not gfserver, why is it so?

```
05/31/2016 05:00:20 PM - breakmer.params - INFO - blat path = /home/xxx/build/bin/blat
05/31/2016 05:00:20 PM - breakmer.params - ERROR - Missing path/executable for gfserver
```

**Comment by ycl6** [comment]
This appears to be caused by the "#" in the breakmer.cfg, where I added some comments that is causing problem. No more error message after remove all "#" and comments. But I thought this issue is resolved in #17 where comments are allowed in config files?

**Comment by mducar** [comment]
Hi,

Sorry for the delay in responding.  The issue with comments in the config file was fixed in the development version but not the stable released version (0.0.6).  Unfortunately the 0.0.7 update contained bugs that we've not had the time to address so the fix for comments never made it into a stable release.

Best,
Matt
